docs/src/content/docs/showcase.mdx

```diff
@@ -0,0 +1,120 @@
+
+<FluidGrid>
+	<Card
+		title="Athena OS"
+		href="https://www.athenaos.org"
+		thumbnail="www.athenaos.org.png"
+	/>
+	<Card
+		title="PubIndexAPI Docs"
+		href="https://docs.pubindexapi.com/"
+		thumbnail="docs.pubindexapi.com.png"
+	/>
+	<Card
+		title="pls"
+		href="https://dhruvkb.github.io/pls"
+		thumbnail="dhruvkb.github.io-pls.png"
+	/>
+	<Card
+		title="capo.js"
+		href="https://rviscomi.github.io/capo.js/"
+		thumbnail="capo.js.png"
+	/>
+	<Card
+		title="Web Monetization API"
+		href="https://webmonetization.org/"
+		thumbnail="webmonetization.org.png"
+	/>
+	<Card
+		title="QBCore Docs"
+		href="https://brycerussell.github.io/qbcore-docs/"
+		thumbnail="brycerussell.github.io-qbcore-docs.png"
+	/>
+	<Card title="har.fyi" href="https://har.fyi/" thumbnail="har.fyi.png" />
+	<Card
+		title="xs-dev docs"
+		href="https://xs-dev.js.org"
+		thumbnail="xs-dev.js.org.png"
+	/>
+	<Card
+		title="Felicity"
+		href="https://felicity.pages.dev/"
+		thumbnail="felicity.pages.dev.png"
+	/>
+	<Card
+		title="NgxEditor"
+		href="https://sibiraj-s.github.io/ngx-editor/"
+		thumbnail="sibiraj-s.github.io-ngx-editor.png"
+	/>
+	<Card
+		title="Astro Error Pages"
+		href="https://astro-error-page-documentation.vercel.app/"
+		thumbnail="astro-error-page-documentation.vercel.app.png"
+	/>
+	<Card
+		title="Terrateam Docs"
+		href="https://terrateam.io/docs"
+		thumbnail="terrateam.io-docs.png"
+	/>
+	<Card
+		title="simple-fm"
+		href="https://simple.arciniega.one"
+		thumbnail="simple.arciniega.one.png"
+	/>
+	<Card
+		title="CommandKit"
+		href="https://commandkit.underctrl.io"
+		thumbnail="commandkit.underctrl.io.png"
+	/>
+</FluidGrid>
+
+See all the [public project repos using Starlight on GitHub](https://github.com/withastro/starlight/network/dependents).
+
+## Community plugins
+
+import { CardGrid, LinkCard } from '@astrojs/starlight/components';
+
+These community tools, plugins, and integrations work alongside Starlight to extend its functionality.
+
+<CardGrid>
+	<LinkCard
+		href="https://www.feelback.dev/blog/new-astro-starlight-integration/"
+		title="FeelBack"
+		description="Add a user feedback system to your docs pages."
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-blog"
+		title="starlight-blog"
+		description="Add a blog to your documentation site."
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-links-validator"
+		title="starlight-links-validator"
+		description="Check for broken links in your Starlight pages."
+	/>
+	<LinkCard
+		href="https://github.com/HiDeoo/starlight-typedoc"
+		title="starlight-typedoc"
+		description="Generate Starlight pages from TypeScript using TypeDoc."
+	/>
+</CardGrid>
+
+## Videos
+
+- Watch Ben and Chris of Astro build [Starwoof](https://starwoof.vercel.app/) with Starlight in real time! [Twitch stream: Astro x Starlight - BLAZING FAST ECO-FRIENDLY NON-GMO ORGANIC docs](https://www.twitch.tv/videos/1841159960)
```

Comment:
Looks like this VOD isn't available anymore
```suggestion
```

Reply:
Ohhhh, I had no idea they could expire like that! Another reason to dislike Twitch 😅 

Good catch — thanks @TheOtterlord!